commands/src/main/java/org/jclouds/karaf/commands/table/internal/ScriptEngineShellTable.java

```diff
@@ -51,9 +47,7 @@ public String evaluate(Object obj, String expression) {
     try {
       scriptEngine.put(getType(), obj);
       result = String.valueOf(scriptEngine.eval(expression));
-    } catch (Exception ex) {
-      //Ignore
-    }
+    } catch (Exception ignored) {}
     return result;
```

Comment:
If we throw the ISE int he constructor, could we assume then that failures here are due to the expressions used to extract the values of the columns, thus, non-fatal errors users can fix by editing the shell.cfg file?